src/plugins/data/public/query/query_string/language_service/language_service.ts

```diff
@@ -57,8 +57,8 @@ export class LanguageService {
     this.languages.set(config.id, config);
   }
 
-  public getLanguage(language: string): LanguageConfig | undefined {
-    return this.languages.get(language);
+  public getLanguage(languageId: string): LanguageConfig | undefined {
```

Comment:
Is this a public API we are changing?

Reply:
changing the parameter variable name shouldn't affect users of this API, and regarding documentation i don't think it's released, didn't find documentation for it